Documentation/get-dotnetcore-windows-dnvm.md

```diff
@@ -0,0 +1,102 @@
+Get .NET Core on Windows with DNVM
+==================================
+
+These instructions will lead you through acquiring .NET Core via the [.NET Version Manager (DNVM)](https://github.com/aspnet/dnvm)  and running a "Hello World" demo on Windows. The instructions use a particular set of paths. You'll need to adjust if you want to use a different set.
+
+These instructions are for .NET Core console apps. If you want to try out ASP.NET 5 on top of .NET Core - which is a great idea - check out the [ASP.NET 5 instructions](https://github.com/aspnet/home).
+
+.NET Core is distributed as NuGet packages. You can acquire it via [NuGet restore](get-coreclr-windows-nuget.md) or the .NET Version Manager (these instructions). Alternatively, you can [build from source](windows-instructions.md). 
+
+You can see the [CoreCLR myget feed](https://www.myget.org/F/dotnet-core) @ the [dotnet-core gallery](https://www.myget.org/gallery/dotnet-core) page. These packages are not yet published to NuGet.org, but that will change soon.
+
+Installing DNVM
+===============
+
+You need DNVM as a starting point. DNVM enables you to acquire a (or multiple) .NET Execution Environment (DNX). DNVM is simply a script, which doesn't depend on .NET. You can install it via a PowerShell command. You can find alternate DNVM install itructions at the [ASP.NET Home repo](https://github.com/aspnet/home).
+
+	@powershell -NoProfile -ExecutionPolicy unrestricted -Command "&{$Branch='dev';iex ((new-object net.webclient).DownloadString('https://raw.githubusercontent.com/aspnet/Home/dev/dnvminstall.ps1'))}"
+
```

Comment:
Ah, I think I had a dirty system (installed dnvm already). I'll re-do with a clean system. I think I recall that you are right. Good catch.